.jenkins/pipelines/standalone/solutions-tests.Jenkinsfile

```diff
@@ -187,4 +187,15 @@ pipeline {
             }
         }
     }
+    post {
```

Comment:
I've added this as these as it's incredibly useful to keep for debug, and to keep track of space usage over time.